flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/pom.xml

```diff
@@ -31,6 +31,14 @@ under the License.
 	<name>Flink : E2E Tests : Common Kafka</name>
 
 	<dependencies>
+<!--		&lt;!&ndash; The SQL Client test uses a Table FileSource, which is not included in the normal table-->
+<!--		dependencies &ndash;&gt;-->
+<!--		<dependency>-->
+<!--			<groupId>org.apache.flink</groupId>-->
+<!--			<artifactId>flink-connector-files</artifactId>-->
+<!--			<version>${project.version}</version>-->
+<!--		</dependency>-->
+
```

Comment:
Remove this?

Reply:
Yes, I put this in because I thought I might have to uncomment it but I was 90% sure that it was not needed.

Reply:
That's why it's in a separate commit. I should have mentioned that before ... 🙈